src/transformers/models/chameleon/image_processing_chameleon.py

```diff
@@ -80,7 +80,7 @@ class ChameleonImageProcessor(BaseImageProcessor):
             Size of the image after resizing. The shortest edge of the image is resized to size["shortest_edge"], with
             the longest edge resized to keep the input aspect ratio. Can be overridden by `size` in the `preprocess`
             method.
-        resample (`PILImageResampling`, *optional*, defaults to 1):
+        resample (`PILImageResampling`, *optional*, defaults to `1`):
```

Comment:
nit - convention is to not have numeric values in code formatting for the docstrings 

```suggestion
        resample (`PILImageResampling`, *optional*, defaults to 1):
```

Reply:
I can revert this, no problem - it was actually changed after running `make fix-copies`